scipy-1.0/paper.tex

```diff
@@ -552,11 +552,11 @@ \section*{Discussion}
 consists of 610 unique contributors of source code, with $>19,000$ commits
 accepted into the code base (GitHub page data).
 
-From the user side, there were 13,096,468 PyPI downloads
-of SciPy during the year 2017\cite{pypinfo}, which is likely a vast
-underestimate of total downloads given that we don't have access to the
-popular Anaconda package manager download statistics. The recommended 
-source for citing SciPy is simply a reference to the website / library
+From the user side, there were 13,096,468 PyPI downloads of SciPy from the
```

Comment:
can remove this mention of PyPI now that it is properly defined / mentioned later in the same sentence